hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/impl/TestContainerPersistence.java

```diff
@@ -267,45 +267,45 @@ public void testAddingBlockToDeletedContainer() throws Exception {
     // so skip check the behaviors related to it.
     assumeFalse(schemaVersion.contains(OzoneConsts.SCHEMA_V3));
 
-    long testContainerID1 = getTestContainerID();
+    long testContainerID = getTestContainerID();
```

Comment:
Pls try to keep changes made to code under review to just address review comments.